backend/research/models.py

```diff
@@ -27,3 +32,132 @@ def can_be_created_by(user):
         return user.is_authenticated
 
     objects = StudyManager()
+
+
+##########################
+# Study Form Submissions #
+##########################
+
+
+class StudyFormSubmissionManager(BaseMRManager):
+    def _viewable_objects(self, user: User):
+        if user.is_privacy_officer or user.is_fetc_member:
+            return self.exclude(status=Statuses.DRAFT)
+        return self.filter(created_by=user)
+
+    def _editable_objects(self, user: User):
+        return self.filter(created_by=user)
+
+
+class StudyFormSubmission(UserFormSubmission):
+    """
+    Subclass of UserFormSubmission, used specifically for submitted forms that
+    relate to a specific study.
+    """
+
+    study = models.ForeignKey(
+        Study, on_delete=models.CASCADE, related_name="form_submissions"
+    )
+
+    @property
+    def status(
+        self,
+    ):
+        return self.status_changes.last().status
+
+
+########################
+# Status Change Object #
+########################
+
+
+class Statuses(models.TextChoices):
+    DRAFT = "DRA"
+    SUBMITTED = "SUB"
+    APPROVED = "APP"
+    REJECTED = "REJ"
+    REVISED = "REV"
+    COMPLETED = "COM"
+
+
+class StatusChange(models.Model):
+    """
+    Object which handles the status of a StudyFormSubmission. These are appended
+    to a StudyFormSubmission as a side effect for certain actions.
+    """
+
+    status = models.CharField(choices=Statuses.choices)
+
+    user_form_submission = models.ForeignKey(
+        "research.StudyFormSubmission",
```

Comment:
Since `StudyFormSubmission` is defined above, can't you simply do the following? That helps with the typing.

```suggestion
        StudyFormSubmission,
```